Calculate.sketchplugin/Contents/Sketch/script.js

```diff
@@ -85,11 +106,17 @@ let showError = function (layer, e) {
   return;
 }
 
+var changedName = function (context) {
```

Comment:
We are not currently using this method, we could remove it safely.

Reply:
This function is fired when a layer name is changed (see manifest.json). it's not implemented yet, but we will need it.

Reply:
Ok!

Reply:
Yeah, you are right, I shouldn't have pushed it. Sorry!